src/inc/corcompile.h

```diff
@@ -1756,6 +1756,11 @@ class ICorCompileInfo
     virtual int GetVersionResilientTypeHashCode(CORINFO_MODULE_HANDLE moduleHandle, mdToken token) = 0;
 
     virtual int GetVersionResilientMethodHashCode(CORINFO_METHOD_HANDLE methodHandle) = 0;
+
+    virtual BOOL EnumMethodsForStub(CORINFO_METHOD_HANDLE hMethod, void** enumerator) = 0;
```

Comment:
I these functions are added here, we are messing up the vtable in any external consumers of this API. It that okay?

Reply:
I don't believe there are any external consumers of this api.